github.go

```diff
@@ -206,15 +203,17 @@ func (base GithubImporter) ImportProject(i Import) error {
 
 func (base GithubImporter) PutProject(repo *github.Repository) error {
 	buildDefinitionStr := base.GetBuildDefiniton(repo.GetOwner().GetLogin(), repo.GetName())
-
 	project, err := base.WharfClient.PutProject(
 		wharfapi.Project{
 			Name:            repo.GetName(),
 			TokenID:         base.Token.TokenID,
 			GroupName:       repo.GetOwner().GetLogin(),
 			BuildDefinition: buildDefinitionStr,
 			Description:     repo.GetDescription(),
-			ProviderID:      base.Provider.ProviderID})
+			AvatarUrl:       *repo.GetOwner().AvatarURL,
+			ProviderID:      base.Provider.ProviderID,
+			GitURL: 		 *repo.GitURL})
```

Comment:
Invalid spacing
```suggestion
			ProviderID:      base.Provider.ProviderID,
			GitURL:          *repo.GitURL})
```

Reply:
Fixed this on a git --fixup but its not showing as resolved here.